overrides/home.html

```diff
@@ -17,7 +17,7 @@
     ]),
     ('For Data Submitters', '#676bc0', '/images/hp-icons-submitter.png', [
         ('View the GDC Data Dictionary', '/Data_Dictionary/viewer/'),
-        ('Search the GDC Data Dictionary', '/Data_Dictionary/gdcmvs/'),
+        ('Search the GDC Data Dictionary', 'https://gdc-mvs.nci.nih.gov/gdc/" target="_blank" rel="noopener noreferrer'),
```

Comment:
In order to have the link open a new tab,  i'm using html-injection at build-time.